var/spack/repos/builtin/packages/py-ase/package.py

```diff
@@ -14,8 +14,11 @@ class PyAse(PythonPackage):
     homepage = "https://wiki.fysik.dtu.dk/ase/"
     url      = "https://pypi.io/packages/source/a/ase/ase-3.13.0.tar.gz"
 
+    version('3.18.0',
+            sha256='39d45f12def2669605bffc82926acfb13a0d0610e6d82740fa316aafa70f97f9')
     version('3.15.0', '65a0143753517c2df157e53bd29a18e3')
     version('3.13.0', 'e946a0addc5b61e5e2e75857e0f99b89')
 
-    depends_on('python@2.6:')
+    depends_on('python@2.6:', when='@:3.15.0')
+    depends_on('python@3.5:', when='@3.18.0:')
     depends_on('py-numpy', type=('build', 'run'))
```

Comment:
Several missing dependencies in the latest version, not sure when they were added:
```python
      install_requires=['numpy', 'scipy', 'matplotlib', 'flask'],               
```

Reply:
Looking at ASE git, 3.14 had these, 3.13 didn't specify any requires. So I guess it's safe to unconditionally require them.

Reply:
Done.